resources/lib/channels/pl/tvp.py

```diff
@@ -5,23 +5,22 @@
 # This file is part of Catch-up TV & More
 
 from __future__ import unicode_literals
-import re
-
-from codequick import Resolver, Script, utils
+import json
```

Comment:
Not sure if it should be defined as a dependency in addon.xml or not ? anyway it seems to be a defaultly or transitively available depenency - missing declaration is not causing bug.